packages/core/src/framebuffer/FramebufferSystem.js

```diff
@@ -285,12 +364,13 @@ export class FramebufferSystem extends System
     }
 
     /**
-     * Update the framebuffer
+     * Updates a bound framebuffer
      *
      * @protected
      * @param {PIXI.Framebuffer} framebuffer
+     * @param {number} [target=this.gl.FRAMEBUFFER] - binding point for `framebuffer`
```

Comment:
mind specifying the 3 types here for clarity?
read / write / framebuffer?

Reply:
I added these in @pixi/constants.